### PR TITLE
Directly submit GNPS job

### PR DESCRIPTION
**Pull request by robinschmid**
Hey Tomas,

I have changed the GNPS export module to have an option to directly submit a GNPS feature based molecular networking job.

The quant table (csv export) and an MS2 mgf are exported and uploaded to GNPS. We are also going to cover this in the GNPS documentation. 

This is a small PR with a huge effect for the MZmine - GNPS connection. I havent actually changed 26 files - most of them only  contain a changed reference.

Cheers
Robin

### CODE REVIEW

File: src/main/java/net/sf/mzmine/modules/peaklistmethods/io/gnpsexport/GNPSExportAndSubmitTask.java [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+ * 
+ * @author Robin Schmid (robinschmid@uni-muenster.de)
+ *
+ */
+public class GNPSExportAndSubmitTask extends AbstractTask {
+  // Logger.
+  private final Logger LOG = Logger.getLogger(getClass().getName());
+
+  /**
+   * This website is still the beta. TODO change later to the final
```
Comment (tomas-pluskal) [inline src/main/java/net/sf/mzmine/modules/peaklistmethods/io/gnpsexport/GNPSExportAndSubmitTask.java]: Is this URL ok for MZmine release?

**Comment by tomas-pluskal** [comment]
Looks great.

For future PRs, please try to separate cosmetic changes (like moving OptionalModuleParameter from net.sf.mzmine.parameters.parametertypes to net.sf.mzmine.parameters.parametertypes.submodules) from new functionality. Cosmetic changes are welcome, too, but it is better to submit them as a separate PR, it makes reviewing easier.

**Comment by robinschmid** [comment]
Yeah I should have reverted the move. In my dev branch there are more classes in this folder.
I usually try to keep it simple, if it is such a small commit.

File: src/main/java/net/sf/mzmine/modules/peaklistmethods/io/gnpsexport/GNPSExportAndSubmitTask.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/net/sf/mzmine/modules/peaklistmethods/io/gnpsexport/GNPSExportAndSubmitTask.java]: This website is going to be the one for a while. 
Later,  when the final website goes live, we can coordinate the transition in MZmine without loss of functionality in any time periode.

**Pull request merged**